configs/rover_configs.yaml

```diff
@@ -37,30 +37,33 @@ core:
 process_discovery:
   # The period of report or keep alive process(second)
   heartbeat_period: ${ROVER_PROCESS_DISCOVERY_HEARTBEAT_PERIOD:20s}
-  # Detect processes in VM mode
-  vm:
-    # Is active the VM mode to detect processes
-    active: ${ROVER_PROCESS_DISCOVERY_VM_ACTIVE:false}
+  # Scan process from linux
+  scanner:
     # The period to detect the process
-    period: ${ROVER_PROCESS_DISCOVERY_VM_PERIOD:3s}
-    finders:
+    period: ${ROVER_PROCESS_DISCOVERY_SCAN_PERIOD:3s}
+    # The process detect mode of scanner, support "REGEX", "AGENT_SENSOR"
+    mode: ${ROVER_PROCESS_DISCOVERY_SCAN_MODE:AGENT_SENSOR}
+    agent:
+      # Supports scanning for how long ago of the active agent
+      recent: ${ROVER_PROCESS_DISCOVERY_AGENT_RECENT:10m}
```

Comment:
`recent` feels a little strange.

Reply:
Why do we need this? And why the value is so long?(10m seems crazy). Rover and SDK are running in the same OS/Linux, I think there wouldn't be a timestamp gap.

Reply:
That means we only collect the process which active how minutes ago. If the process has been dead(no keepalive again), so we don't need to collect them. 

> And why the value is so long?

It's decided by how long is the process to keep alive. I could update to `1m`.

Reply:
OK, then this name is not proper.  `period` is there for a rover, should this be `agentProcessStatusRefreshPeriod`?

`recent` has no specific meaning in this case. `recent` is describing what is happening now, not a concept of timestamp gap.

Reply:
Update to `process_statusr_efresh_period`, and `1m` as default.

Reply:
`process_statusr_efresh_period` typo? of `statusr`?

Reply:
Oh, my mistake. It's `process_status_refresh_period`